hca-to-scea/helpers/multitab_excel_to_single_txt.py

```diff
@@ -1,9 +1,35 @@
 from pathlib import Path
 import numpy as np
 import pandas as pd
+import sys
+import copy
 
 from helpers import utils
 from helpers import get_protocol_map
+from helpers import cell_lines
+
+def remove_unused_protocols(xlsx_dict):
+
+    xlsx_dict_tmp = copy.deepcopy(xlsx_dict)
+
+    '''Delete unused tabs.'''
+    for tab in xlsx_dict.keys():
+        if "protocol" in tab:
+            if xlsx_dict[tab].empty:
+                del xlsx_dict_tmp[tab]
+
+    xlsx_dict_tmp2 = copy.deepcopy(xlsx_dict_tmp)
+
```

Comment:
This bit should be a function of its own. You could name it like the comment.
We need to keep functions in the same level of abstraction and details, and not to overburden the reader with too many implementation details.